crates/llm-local/Cargo.toml

```diff
@@ -6,30 +6,32 @@ edition = { workspace = true }
 
 [dependencies]
 anyhow = "1.0"
-candle = { git = "https://github.com/huggingface/candle", rev = "b80348d22f8f0dadb6cc4101bde031d5de69a9a5", package = "candle-core" }
-candle-nn = { git = "https://github.com/huggingface/candle", rev = "b80348d22f8f0dadb6cc4101bde031d5de69a9a5" }
+candle = { git = "https://github.com/huggingface/candle", package = "candle-core", features = [
```

Comment:
Why are we still using the git dependency instead of crates.io?

Reply:
it is. due to use using `llama::LlamaEosToks` which is not part of the last release available on crates.io.

Reply:
Can we keep a `rev` here so that we're not relying on Cargo.lock for the right version?

Reply:
Added a `rev` tag.